tests/resources/conformance_suites/nl_nt18/br_kvk/2-04-invalid-period.xbrl

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I wonder if it would be easier to maintain if we only had one copy of the files for the latest version (e.g., nt18 for now) and then programmatically transformed into older versions (e.g., nt17 and nt16), with a small bit of test metadata somewhere describing tests that only apply to certain versions.

Reply:
That's how this was generated behind the scenes, I plan on adding the template code to an `Arelle/confomrance-suites` repository that we are in the process of setting up.